benchmarks/pca/solvers/scikit_learn.py

```diff
@@ -0,0 +1,77 @@
+from importlib.metadata import version
+
+from benchopt import BaseSolver, safe_import_context
+from benchopt.stopping_criterion import SingleRunCriterion
+
+with safe_import_context() as import_ctx:
+    from sklearn.decomposition import PCA
+
+
+class Solver(BaseSolver):
+    name = "scikit-learn"
+    requirements = ["scikit-learn"]
+
+    parameters = dict(
+        svd_solver=["full", "arpack", "randomized"],
+        power_iteration_normalizer=["QR", "LU", "none"],
```

Comment:
I would rather not bench all those combinations.

For randomized we could set `power_iteration_normalizer="LU"` when using numpy and `power_iteration_normalizer="QR"` when using Array API (PyTorch).